dali/operators/reader/loader/video_loader.cc

```diff
@@ -24,7 +24,8 @@
 #include <limits>
 #include <sstream>
 
-inline int gcd(int a, int b) {
+
+static inline int gcd(int a, int b) {
```

Comment:
Why? An inline function has internal linkage, no need for static.

Reply:
Done